On `x64-osx`:
```
CMake Error at cmake/c4Project.cmake:1661 (message):
  not implemented
Call Stack (most recent call first):
  CMakeLists.txt:75 (c4_install_exports)
```
Is this expected?

--- a/ports/c4core/CONTROL
+++ b/ports/c4core/CONTROL
@@ -2,4 +2,4 @@ Source: c4core
 Version: 2020-04-12
 Homepage: https://github.com/biojppm/c4core
 Description: Library of low-level C++ utilities 
-Supports: !(arm|arm64)
+Supports: !(arm|arm64|osx)

--- a/ports/c4core/portfile.cmake
+++ b/ports/c4core/portfile.cmake
@@ -68,8 +68,8 @@ file(READ "${CURRENT_PACKAGES_DIR}/share/c4core/c4coreConfig.cmake" _contents)
 string(REGEX REPLACE [[[ \t\r\n]*"\${PACKAGE_PREFIX_DIR}[\./\\]*"]] [["${PACKAGE_PREFIX_DIR}/../.."]] _contents "${_contents}")
 file(WRITE "${CURRENT_PACKAGES_DIR}/share/c4core/c4coreConfig.cmake" "${_contents}")
 
-# Fix missing header (see c4/error.hpp)
-file(RENAME "${CURRENT_PACKAGES_DIR}/include/c4/extern/debugbreak" "${CURRENT_PACKAGES_DIR}/include/debugbreak")
+# Fix path to header
+vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/include/c4/error.hpp" "<debugbreak/debugbreak.h>" "\"extern/debugbreak/debugbreak.h\"")
 
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
 

--- a/ports/c4core/portfile.cmake
+++ b/ports/c4core/portfile.cmake
@@ -1,5 +1,9 @@
 vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
-vcpkg_fail_port_install(ON_ARCH "arm" "arm64")
+
+vcpkg_fail_port_install(
+    ON_ARCH "arm" "arm64"
+    ON_TARGET "OSX"
+)
 
 # Get c4core src
 vcpkg_from_github(

--- a/scripts/ci.baseline.txt
+++ b/scripts/ci.baseline.txt
@@ -1900,7 +1900,7 @@ zkpp:x64-windows-static=fail
 zkpp:arm64-windows=fail
 zkpp:x64-uwp=fail
 zkpp:arm-uwp=fail
-boost-asio:x86-windows=ignore
-argumentum:x86-windows=ignore
 c4core:arm-uwp=fail
 c4core:arm64-windows=fail
+c4core:x64-osx=fail
+

